src/kernel/capinv.c

```diff
@@ -644,13 +623,16 @@ cap_thd_op(struct cap_thd *thd_cap, struct thread *thd, struct pt_regs *regs,
 		arcv_cap = (struct cap_arcv *)captbl_lkup(ci->captbl, arcv);
 		if (!CAP_TYPECHK_CORE(arcv_cap, CAP_ARCV)) return -EINVAL;
 
-		rcvt = arcv_cap->thd;
+		rcvt     = arcv_cap->thd;
```

Comment:
This is a grey area in the CSG.  General rule:  if it looks better, do it.  That argument could be made here.  The converse argument could be made as well.  The author wins these arguments ;-)